appcelerator/titanium_mobile
# fix(android): TabGroup bar background color is wrongly transparent on Android 4.4 as of 8.0.0

**JIRA:**
- https://jira.appcelerator.org/browse/TIMOB-27117
- https://jira.appcelerator.org/browse/TIMOB-27100

**Summary:**
- [TIMOB-27117](https://jira.appcelerator.org/browse/TIMOB-27117) Fixed bug where `TabGroup` bar background color is wrongly transparent on Android 4.4 as of Titanium 8.0.0.
- [TIMOB-27100](https://jira.appcelerator.org/browse/TIMOB-27100) Fixed 8.1.0 regression where tab text color is wrongly transparent.
  * This was bug was never in a released version. Caught before release.
- Modified `TabGroup` to fetch bar background color and text color from `ActionBar` style/theme for backward compatibility.

**Test:**
1. Set up a Classic Titanium app with the code attached to [TIMOB-27117](https://jira.appcelerator.org/browse/TIMOB-27117).
2. Build and run on an Android 4.4 device.
3. Verify that the tab labels appear.
4. Verify that the tab bar's background is dark gray like the title bar.
